resources/views/form.blade.php

```diff
@@ -33,3 +33,31 @@
         </div>
     </graphql>
 </lazy>
+
+{{-- TODO: Fix the above code and remove this static HTML --}}
```

Comment:
So what's the problem?

Reply:
Not sure @jbclaudio said he would like to look into it